DevEdu.API/DevEdu.Business.Tests/RatingServiceTests.cs

```diff
@@ -43,7 +43,10 @@ public void AddStudentRating_StudentRatingDto_AuthorUserId_StudentRatingCreated(
             var groupId = group.Id;
             var student = inputStudentRatingDto.User;
             var studentId = student.Id;
-            var authorUserId = 1;
+            var authorUserInfo = new UserIdentityInfo { 
+                UserId = 1,
+                Roles = new List<Role> { Role.Teacher } 
+            };

```

Comment:
please make a method that will return such objects
userId and userRole will be the parameters of this method